.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+on: [push, pull_request]
+
+jobs:
+  run-conformance:
+    runs-on: ubuntu-latest
+    name: Validate conformance test framework
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
+
+    - name: Vendor dependencies
+      run: "go mod vendor"
```

Comment:
Why vendor? (Fine to do, but I don't think it will be used by Go 1.13? The `gofmt` invocation would be simpler without (could also move the `gofmt` call above the `go mod vendor`).

Reply:
Good points :) thanks!